[2025-05-15T08:25:32.151+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-15T08:25:32.180+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 06_python_xcom_prac_dag.python_task1 scheduled__2025-05-14T01:30:00+00:00 [queued]>
[2025-05-15T08:25:32.191+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 06_python_xcom_prac_dag.python_task1 scheduled__2025-05-14T01:30:00+00:00 [queued]>
[2025-05-15T08:25:32.191+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 4
[2025-05-15T08:25:32.210+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): python_task1> on 2025-05-14 01:30:00+00:00
[2025-05-15T08:25:32.231+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', '06_python_xcom_prac_dag', 'python_task1', 'scheduled__2025-05-14T01:30:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/06_xcom_practice_dag.py', '--cfg-path', '/tmp/tmpxl05uvhl']
[2025-05-15T08:25:32.229+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3265) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-15T08:25:32.234+0000] {standard_task_runner.py:72} INFO - Started process 3266 to run task
[2025-05-15T08:25:32.233+0000] {standard_task_runner.py:105} INFO - Job 17: Subtask python_task1
[2025-05-15T08:25:32.314+0000] {task_command.py:467} INFO - Running <TaskInstance: 06_python_xcom_prac_dag.python_task1 scheduled__2025-05-14T01:30:00+00:00 [running]> on host b13e4d6f8681
[2025-05-15T08:25:32.457+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='haein@***.com' AIRFLOW_CTX_DAG_OWNER='haein' AIRFLOW_CTX_DAG_ID='06_python_xcom_prac_dag' AIRFLOW_CTX_TASK_ID='python_task1' AIRFLOW_CTX_EXECUTION_DATE='2025-05-14T01:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-14T01:30:00+00:00'
[2025-05-15T08:25:32.460+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-15T08:25:32.510+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-15T08:25:32.530+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-15T08:25:32.531+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=06_python_xcom_prac_dag, task_id=python_task1, run_id=scheduled__2025-05-14T01:30:00+00:00, execution_date=20250514T013000, start_date=20250515T082532, end_date=20250515T082532
[2025-05-15T08:25:32.620+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-15T08:25:32.674+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-15T08:25:32.676+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
